src/java/com/vaklinov/zcashui/ZCashClientCaller.java

```diff
@@ -95,6 +82,9 @@ public WalletCallException(String message, Throwable cause)
 		}
 	}
 
+	// ZenCash Advanced settings variables
+	public String commands = "";
+	public boolean runOnce = applyZendCommands();
```

Comment:
It should be better to initialize both fields in constructor as it is not clear where commands is initialized (see line 143 String[] zend_commands = commands.isEmpty() ...)